topics/releases.md

```diff
@@ -1,144 +1,109 @@
-it is among the most critical pieces of a software stack.
+Valkey is usually among the most critical pieces of a software stack.
+For this reason, Valkey's release cycle prioritizes highly stable releases at the cost of slower release cycles.
 
-For this reason, Valkey's release cycle is such that it ensures highly stable
-releases, even at the cost of slower cycles.
+All Valkey releases are published in the [Valkey GitHub repository](https://github.com/valkey-io/valkey/releases).
 
-New releases are published in the [Valkey GitHub repository](https://github.com/valkey-io/valkey/releases).
-
-## Release cycle
-
-A given version of Valkey can be at three different levels of stability:
-
-* Unstable
-* Release Candidate
-* Stable
-
-### Unstable tree
-
-The unstable version of Valkey is located in the `unstable` branch in the
-[Valkey GitHub repository](https://github.com/valkey-io/valkey).
-
-This branch is the source tree where most of the new features are under
-development. `unstable` is not considered production-ready: it may contain
-critical bugs, incomplete features, and is potentially unstable.
-
-However, we try hard to make sure that even the unstable branch is usable most
-of the time in a development environment without significant issues.
-
-### Release candidate
-
-New minor and major versions of Valkey begin by branching off the `unstable`
-branch. The branch name is the target release on the form *major.minor*.
-Subsequent patch releases are made on the same branch.
-
-For example, when Valkey 7.2.5 was released, the release was made on the `7.2`
-branch, which had been branched off from `unstable` earlier.
+## Versioning
 
-Bug fixes and new features that can be stabilized during the release's time
-frame are committed to the unstable branch and backported to the release
-candidate branch. The `unstable` branch may include additional work that is not
-a part of the release candidate and scheduled for future releases.
+Valkey stable releases will generally follow `major.minor.patch` [semantic versioning schema](https://semver.org/).
+We follow semantic versioning to provide explicit guarantees regarding backward compatibility.
 
-The first release candidate, or RC1, is released once it can be used for
-development purposes and for testing the new version. At this stage, most of
-the new features and changes the new version brings are ready for review, and
-the release's purpose is collecting the public's feedback.
+When discussing compatibility, we refer to the following API contracts:
 
-Subsequent release candidates are released every three weeks or so, primarily
-for fixing bugs. These may also add new features and introduce changes, but at
-a decreasing rate and decreasing potential risk towards the final release
-candidate.
+1. Valkey commands including their inputs, outputs, and defined behavior
+2. The functions and APIs that can be executed from a Lua script
+3. The RDB version
```

Comment:
AOF is the other persistence mechanism we support. It has its own on-disk format so it would be worth mentioning here too.

Reply:
Yeah, good point